oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,138 @@
++-----------------+----------------------------------------------------------------+
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | <TODO>                                                         |
++-----------------+----------------------------------------------------------------+
+
+Context
+-------
+
+In the Open edX system, course content authoring occurs in the Studio tool, which focuses mainly on authoring of
+course structure and content within a **Course Run**. A Course Run is discoverable on the edX marketing site via
+the context of a **Catalog Course**. The **Catalog Course** data authoring needs a good user experience.
+
+Today, Catalog Course metadata authoring occurs in multiple places, including the Studio, Ecommerce, and
+Marketing sites. We introduced a new tool, called Publisher, to ease the Catalog Course metadata 
+authoring process. The Publisher tool includes an inflexible workflow engine for the process of provisioning
+a course. Currently, this course publish cycle can take up to fifteen days in order to provision 
+a new course. 
+
+We also run into data synchronization issues between the separate services since there are multiple sources of truth and
+multiple writers of the relevant metadata.
+
+Decisions
+---------
+
+* We will no longer embed workflow into the core system. Rather, we will allow for a self-service workflow tool that will reduce the provisioning time.
+
+* Conceptually, we will separate the notion of Course within the edX system in the following ways:
+
+  * **Course Learning Content**, a course-team created package of learning material, with its weekly and sub-weekly structure
+    (i.e., Sections, Subsections, Units, etc) and learning blocks (i.e., leaf xBlocks), tied to specific dates of offering.
+     
+    * This is what we call **(Studio) Course Run**, for the time being.
+    * It is bound to the organization that authors the content.
+    * Note: In the future, this concept may be further separated when we support content reuse across "Course Runs".
+      At that time, we may distinguish between:
+
+      * **Reusable Learning Content** (not tied down to specific dates nor structure) and
+      * **Packaged Learning Content** (an "instantiation" of a "Reusable Learning Content" with structure, dates, and 
+        configuration).
+
+  * **Discoverable Course**, a learner-facing representation of a course that entices enrollment with marketing information
+    such as the course's price and credit/degree outcomes.
+
+    * This is what we call **Catalog Course**, for the time being.
+    * It is bound to a specific marketing site.
```

Comment:
What does this mean?

Reply:
This is not true for Openedx today. OpenedX's default settings use these courses for discovery on the home page.